pkg/cloudprovider/providers/gce/gce_disks.go

```diff
@@ -54,6 +55,8 @@ const (
 	replicaZoneURITemplateSingleZone = "%s/zones/%s" // {gce.projectID}/zones/{disk.Zone}
 )
 
+var DiskNumberLimit = []int{16, 32, 64, 128}
```

Comment:
Can we make this a map instead? with key = num cpus, and value = pd limit?

Reply:
I am not sure about how many cpus possible support

Reply:
Or at least define an enum to represent the index?

Reply:
added enum